src/test/java/core/basesyntax/service/DataReaderTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.DataReaderImpl;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class DataReaderTest {
+    private static final String APPLES_ORANGES = "src/test/resources/applesOranges.csv";
+    private DataReader dataReader;
+
+    @Before
+    public void setUp() {
+        dataReader = new DataReaderImpl();
+    }
+
+    @Test
+    public void readData_Ok() {
+        List<String> dataList = dataReader.readData(APPLES_ORANGES);
+        String actualResult = dataList.stream().collect(Collectors.joining(System.lineSeparator()));
+        String expectedResult = "b,apple,35" + System.lineSeparator()
```

Comment:
create list with expected strings manually and compare it with the one you got from `readData()`

Reply:
or create private helper method that will read the same file and compare results